app/build.gradle

```diff
@@ -35,10 +36,13 @@ android {
     }
 }
 
+
 dependencies {
 
-    def lifecycle_version = "2.6.0-alpha01"
+    implementation 'androidx.navigation:navigation-fragment-ktx:2.5.0'
```

Comment:
you can also create a version property for the navigation dependencies, similar to the lifecycle ones